spacewalk/certs-tools/rhn_bootstrap_strings.py

```diff
@@ -276,23 +278,19 @@ def getRegistrationStackSh(saltEnabled):
 }}
 
 function clean_up_old_salt_repos() {{
-  local salt_minion_channels_file="susemanager:channels.repo"
-
-  if [ -f "$1/$salt_minion_channels_file" ] ; then
-    echo "Removing $1/$salt_minion_channels_file"
-    rm -f "$1/$salt_minion_channels_file"
+  if [ -f "$1" ] ; then
+    echo "Removing $1"
+    rm -f "$1"
   fi
 }}
 
 function clean_up_old_repos() {{
-  local suse_os_repos_path="/etc/zypp/repos.d"
-  local redhat_os_repos_path="/etc/yum.repos.d"
-
-  clean_up_old_salt_repos $suse_os_repos_path
-  clean_up_old_salt_repos $redhat_os_repos_path
+  clean_up_old_salt_repos "/etc/zypp/repos.d/susemanager:channels.repo"
+  clean_up_old_salt_repos "/etc/yum.repos.d/susemanager:channels.repo"
+  clean_up_old_salt_repos "/etc/apt/sources.list.d/susemanager:channels.list"
 
-  clean_up_old_trad_repos $suse_os_repos_path
-  clean_up_old_trad_repos $redhat_os_repos_path
+  clean_up_old_trad_repos "/etc/zypp/repos.d"
+  clean_up_old_trad_repos "/etc/yum.repos.d"
```

Comment:
Don't we need this cleanup for ubuntu clients as well?

Reply:
Ah, just realized that ubuntu clients always will be salt minions, not traditional clients. So no cleanup needed.

Reply:
Right. There's shouldn't be any traditional tools on an Ubuntu system.